src/std/array.ab

```diff
@@ -17,11 +19,13 @@ pub fun array_search(array, value): [Num] {
     return result
 }
 
+/// Check if the value is in the array
 pub fun in_array(array, value): Bool {
     let result = array_first_index(array, value)
     return result >= 0
 }
 
+/// Check if the value is in the array
```

Comment:
Why do we have both of those functions?

Reply:
Do you mean `includes` and `in_array`?

To me they do the same things and probably initially there was one and later the other one it was added.
I guess that we can remove `includes` as the other one is better.

Reply:
I prefer the name `includes` over `in_array` as it's more natural